reference/api/search.mdx

```diff
@@ -1118,3 +1122,17 @@ Use a custom vector to perform a search query. Must be an array of numbers corre
 <Capsule intent="warning">
 Meilisearch will return an error if you use `vector` before activating your instance's `vectorStore` and [configuring a custom embedder](/reference/api/settings#embedders-experimental).
 </Capsule>
+
+### Display `_vectors` in response (experimental)
+
+**Parameter**: `retrieveVectors`<br />
+**Expected value**: `true` or `false`<br />
+**Default value**: `false`
+
+Return document vector data in AI-powered searches.
+
+This parameter has no effect on queries that do not include `vector` or `hybrid`.
```

Comment:
```suggestion
```

This is incorrect. If the document contains vector data, setting `retrieveVectors` to true will retrieve them, regardless of the kind of search used.